frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -119,56 +119,48 @@ const LogsTableInner = ({
 	const queryTerms = parseLogsQuery(query)
 	const [expanded, setExpanded] = useState<ExpandedState>({})
 
-	const columns = React.useMemo<ColumnDef<LogEdge>[]>(
-		() => [
-			{
-				accessorKey: 'node.timestamp',
-				cell: ({ row, getValue }) => (
-					<Box
-						flexShrink={0}
-						flexDirection="row"
-						display="flex"
-						alignItems="flex-start"
-						gap="6"
-					>
-						{row.getCanExpand() && (
-							<Box
-								display="flex"
-								alignItems="flex-start"
-								cssClass={styles.expandIcon}
-							>
-								{row.getIsExpanded() ? (
-									<IconExpanded />
-								) : (
-									<IconCollapsed />
-								)}
-							</Box>
-						)}
-						<LogTimestamp timestamp={getValue() as string} />
```

Comment:
We no longer need `as string` since getValue() knows the type:
<img width="712" alt="Screenshot 2023-06-26 at 10 06 12 PM" src="https://github.com/highlight/highlight/assets/58678/6be5b2cd-9e10-4038-bd9c-fd07c061fb64">